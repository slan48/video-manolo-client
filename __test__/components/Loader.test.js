import React from 'react';
import { mount } from 'enzyme';
import Loader from "../../components/Loader";
import { create } from 'react-test-renderer';

describe('<Loader />', function () {
  test('Loader component render', () => {
    const loader = mount(<Loader />);
    expect(loader.length).toEqual(1);
  })

  test('Loader Snapshot', () => {
    const loader = create(
      <Loader />
    );
    expect(loader.toJSON()).toMatchSnapshot();
  })
});
