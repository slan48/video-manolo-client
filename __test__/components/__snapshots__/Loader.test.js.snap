// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Loader Snapshot 1`] = `
<div
  className="jsx-1940039920"
>
  <div
    className="jsx-1940039920 lds-grid"
  >
    <div
      className="jsx-1940039920"
    />
    <div
      className="jsx-1940039920"
    />
    <div
      className="jsx-1940039920"
    />
    <div
      className="jsx-1940039920"
    />
    <div
      className="jsx-1940039920"
    />
    <div
      className="jsx-1940039920"
    />
    <div
      className="jsx-1940039920"
    />
    <div
      className="jsx-1940039920"
    />
    <div
      className="jsx-1940039920"
    />
  </div>
</div>
`;
